### PR TITLE
Named Schedules

### PR DESCRIPTION
**Pull request by Voark**
Add scheduleName to schema and set default values. Also updated local port to what it should've been.

### CODE REVIEW

**Comment by Voark** [comment]
https://github.com/icssc/AntAlmanac/pull/375

File: server.local.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const setup = require('./server')
-const port = 3000
```
Comment (ChaseC99) [inline server.local.js:2]: Why the port change?

File: models/User.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ const User = mongoose.Schema({
                 scheduleIndices: [Number],
             },
         ],
+        scheduleNames: {type: [String], default: ['Schedule 1', 'Schedule 2', 'Schedule 3', 'Schedule 4']},
```
Comment (ChaseC99) [inline models/User.js:14]: By default, the frontend starts with on `['Schedule 1']`. Should this match that?

File: server.local.js (hunk 1 above)
Comment (Voark) [reply server.local.js:2]: image

Comment (Voark) [reply server.local.js:2]: I don't remember why it was 3000 initially, I probably just wasn't paying attention and never ran the backend locally anwyays.

**Review by Douglas-Hong — APPROVED** [review]
Looks good to me!

**Pull request merged**